templates/index.html

```diff
@@ -39,6 +39,8 @@ <h1>laniakea books</h1>
                 <a href="/lets-chat/">Let’s chat</a>
                 <a href="/how-it-works/">How it works</a>
                 <a href="/philosophy/">Philosophy</a>
+                <a href="/handbook/">Handbook</a>
+                <a href="/handbook-clustered/">Handbook clustered</a>
```

Comment:
Let's remove them from the navbar, it's getting a bit crowded and the handbook is not that important I think!